poms/bom/pom.xml

```diff
@@ -1446,7 +1446,13 @@
             <dependency>
                 <groupId>org.apache.camel</groupId>
                 <artifactId>camel-optaplanner</artifactId>
-                <version>${camel.version}</version>
+                <version>3.6.0-SNAPSHOT</version>
```

Comment:
Is the versions hardcoded or dont the branch use camel.version for the SNAPSHOT also to track master ?

Reply:
@davsclaus it's a mistake. I'll correct that!!